### DISCUSSION TITLE
blog/blog-write-for-others-or-yourself

### DISCUSSION THREAD

**Discussion by giscus** [Pathsis/2750-words#18] [Announcements]
# blog/blog-write-for-others-or-yourself

写给自己和写给别人很不一样。每个人的人生都与众不同，而每个人都多少有些窥私欲。我看别人的博客，会觉得津津有味，因为我不熟悉他的生活，他提供的信息超出了我对他认识的100%。但如果他看自己的博客，却未必会有我这样的感觉，因为他很熟悉自己的生活，他提供的信息也许只反映了自己认识的1%。

https://rercel.com/blog/blog-write-for-others-or-yourself

**Comment by seeyousuperman** [comment]
你写的真是篇篇直击人心，很厉害。
